packages/react/src/button/styles.ts

```diff
@@ -95,12 +95,14 @@ export const loadingSize = {
 
 export function buttonStyles({
 	block,
-	variant,
+	focusRingFor = 'keyboard',
 	size,
+	variant,
 }: {
 	block: boolean;
-	variant: ButtonVariant;
 	size: ButtonSize;
+	variant: ButtonVariant;
+	focusRingFor?: BoxProps['focusRingFor'];
```

Comment:
🔤

Reply:
I stuck it at the end because it's optional but I think moving the default to `Button` can make this required. Unless we're considering that a breaking change because it's exported by the package?

Reply:
Hmm it breaks too many existing things to be worthwhile.

Reply:
So it is. Perhaps have it as the default in both then? It's duplication (sorry, SonarCloud), but it makes usage clearer.

I was going to suggest that `Button` should be changed to be made of `Box` instead so we could remove all this `focusFor` work,  but since `buttonStyles` is used in 3 other places, it will now go cry in the corner instead